.github/actions/optimized_build/action.yaml

```diff
@@ -29,17 +29,20 @@ runs:
     - name: "Check for WASM binaries larger than allowed limit"
       shell: "sh"
       run: |
-        find_files_cmd="find \"artifacts/\" -size +1500k | wc -l"
+        large_files=`find "artifacts/" -size +3M`
 
-        large_files_count=$(eval "${find_files_cmd}")
+        large_files_count=`wc -l < EOF
+          ${large_files}
+          EOF`
```

Comment:
what about
large_files=`find "artifacts/" -size +3M | wc -l`

Reply:
`` is the same as `eval`

Reply:
I want to collect the file names too, which is half the point of this PR.